src/common/libflux/event.c

```diff
@@ -22,6 +22,68 @@
 #include "rpc.h"
 #include "message.h"
 
+flux_future_t *flux_event_subscribe_ex (flux_t *h,
+                                        const char *topic,
+                                        int flags)
+{
+    if (!h || !topic) {
+        errno = EINVAL;
+        return NULL;
+    }
+    return flux_rpc_pack (h,
+                          "event.subscribe",
+                          FLUX_NODEID_ANY,
+                          flags,
+                          "{s:s}",
+                          "topic", topic);
+}
+
+int flux_event_subscribe (flux_t *h, const char *topic)
+{
+    flux_future_t *f;
+
+    if (h && (flux_flags_get (h) & FLUX_O_TEST_NOSUB))
+        return 0;
+    if (!(f = flux_event_subscribe_ex (h, topic, 0))
+        || flux_future_get (f, NULL) < 0) {
+        flux_future_destroy (f);
+        return -1;
+    }
+    flux_future_destroy (f);
+    return 0;
+}
+
+flux_future_t *flux_event_unsubscribe_ex (flux_t *h,
+                                          const char *topic,
+                                          int flags)
+{
+    if (!h || !topic) {
+        errno = EINVAL;
+        return NULL;
+    }
```

Comment:
similar to above, if `flux_rpc_pack()` also checks args we're checking twice here (no harm though)